samples/SimpleDataSource/SimpleCustomDataSource.cs

```diff
@@ -41,6 +41,65 @@ public class SimpleCustomDataSource
     {
         private IApplicationEnvironment applicationEnvironment;
 
+        //
+        // Provides information about this Custom Data Source, such as the author,
+        // a project link, licensing, etc. This information can be used by tools
+        // to display "About" information for your Custom Data Source. For example,
+        // Windows Performance Analyzer (WPA) uses this information for Help->About.
+        //
+
+        public override CustomDataSourceInfo GetAboutInfo()
```

Comment:
>CustomDataSourceInfo [](http://example.com/codeflow?start=24&length=20)

I think this is being renamed to ProcessingSourceInfo. Is that class avalable yet?

Reply:
We are, this is consistent with what is currently there. I will update in a separate change.

Reply:
https://github.com/microsoft/microsoft-performance-toolkit-sdk/issues/87